tests/kafkatest/services/zookeeper.py

```diff
@@ -23,6 +23,7 @@
 
 
 class ZookeeperService(Service):
+    KAFKA_OPTS = ""
```

Comment:
Is there a reason you haven't followed the pattern in `KafkaService` where each instance has its own `SecurityConfig`?

I believe this would let you update `zk.security_config` and then have `start_node` take care of fetching `kafka_opts` (instead of having to set manually within a test method), and also take care of generating the `jaas_digest` thing on the zk nodes (instead of doing so manually within the test method)